Generator.Domain/Entities/Choice.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Generator.Domain.Entities
+{
+    public class Choice
+    {
+        public Choice(Guid pictureAId, Guid pictureBId, Guid userChoiceId)
+        {
+            PictureAId = pictureAId;
+            PictureBId = pictureBId;
+            UserChoiceId = userChoiceId;
+        }
+        public Guid PictureAId { get; }
+
+        public string PictureA { get; set; }
```

Comment:
dalej nie wiem czy trzymanie tego w stringu i w taki sposob jest dobrym pomyslem

Reply:
i w sumie jesli to jest entity od wyboru to po co on przechowuje obrazek?